Dockerfile

```diff
@@ -14,6 +14,10 @@ WORKDIR /root/kava
 COPY ./go.mod ./
 COPY ./go.sum ./
 
+RUN echo 3
+RUN mkdir -p /Users/yevheniishcherbina/go/src/github.com/Kava-Labs
```

Comment:
Recommend making this a variable vs hard-coded, for example

https://stackoverflow.com/questions/40902445/using-variable-interpolation-in-string-in-docker

Reply:
It will be removed, it's temporary